### PR TITLE
Support profile to export environments

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]

File: internal/components/setup/compose.go [hunk 1]
```diff
@@ -67,7 +68,17 @@ func ComposeSetup(e2eConfig *config.E2EConfig) error {
 		return fmt.Errorf("bind wait ports error: %v", err)
 	}
 
-	execError := compose.WithCommand([]string{"up", "-d"}).Invoke()
+	// build command
+	cmd := make([]string, 0)
+	if e2eConfig.Setup.InitSystemEnvironment != "" {
+		profilePath := util.ResolveAbs(e2eConfig.Setup.InitSystemEnvironment)
+		cmd = append(cmd, "--env-file", profilePath)
+		util.ExportEnvVars(profilePath)
```
Comment (kezhenxu94) [inline internal/components/setup/compose.go:76]: You've already added it to the command option, why you need to export it ?

Comment (mrproliu) [reply internal/components/setup/compose.go:76]: It's goods to let `setup.steps` or `verify.query` commands could be using these environments too.

**Comment by kezhenxu94** [comment]
Update the doc please. Otherwise LGTM

**Comment by mrproliu** [comment]
> Update the doc please. Otherwise LGTM

The document has been updated, should I miss anything?

File: docs/en/setup/Configuration-File.md [hunk 2]
```diff
@@ -48,14 +48,16 @@ The `KinD` environment follow these steps:
...
+  file: path/to/compose.yaml            # Specified docker-compose file path
+  timeout: 1200                         # Timeout second
+  init-system-environment: path/to/env  # Import environment file
+  steps:                                # Customize steps for prepare the environment
+    - name: customize setups            # Step name
+      command: command lines            # Use command line to setup 
 ```
 
 The `docker-compose` environment follow these steps:
+1. Import `init-system-environment` file for help build service and execute steps.
```
Comment (wu-sheng) [inline docs/en/setup/Configuration-File.md:60]: I think should describe what format this file is.

Comment (mrproliu) [reply docs/en/setup/Configuration-File.md]: been added. Please take a look :)

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**